docs/phase-2-migration/deployment-guide.md

```diff
@@ -1,4 +1,4 @@
-# Process for Deploying a Finder to V2
+## Deployment guide
 
 After following this process, a specified finder would be served from v2 `specialist-publisher` as opposed to v1.
```

Comment:
(Can't comment on stuff not changed in this PR)

Rather than running `Sidekiq::Queue.all.first.size` a lot, link to the documentation for sidekiq monitoring in the ops manual